react/AddressForm.tsx

```diff
@@ -108,30 +109,38 @@ const AddressForm: StorefrontFunctionComponent = () => {
 
   const parseLineFragment = (fragment: LineFragment) => {
     const field = fields[fragment.name as keyof Fields]
+
     if (
       !field ||
       ignoredFields.has(fragment.name) ||
       address[fragment.name] == null
-    )
+    ) {
       return null
+    }
 
-    if (hasWithoutNumberOption(field.label))
-      return <NumberOption showCheckbox />
+    const style = styleRules[field.label]
+
+    let fragmentComponent = undefined
```

Comment:
nit: rename this to `fragmentElement`, since it's won't be a component, but an element